package.json

```diff
@@ -51,8 +51,15 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "babel": "^5.8.21",
+    "babel-cli": "^6.3.17",
+    "babel-core": "^6.3.21",
     "babel-eslint": "^4.1.4",
+    "babel-istanbul": "^0.5.9",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "babel-register": "^6.3.13",
+    "babel-runtime": "^6.3.19",
```

Comment:
It seems very unfortunate to require `babel-runtime` in a reusable module. Why is this necessary?